chacra/async/rpm.py

```diff
@@ -73,16 +73,24 @@ def create_rpm_repo(repo_id):
         except OSError:
             logger.exception('could not symlink')
 
-    for d in repo_dirs:
-        # this prevents RPM packages that are larger than 2GB (!!!) from
-        # causing the database to fail to store the size and subsequently make
-        # the package uninstallable. Ideally, this types of flag options should
-        # be configurable
-        subprocess.check_call(['createrepo', '--no-database', d])
+    _do_createrepo(paths['absolute'], repo_dirs, repo.distro)
```

Comment:
_do_createrepo => _create_rpm_repo

Reply:
really? why?

Reply:
as I understand you've created this function in order to isolate test point of actual create repository for the 'public' method `create_rpm_repo` in order to provide a functional test for it, it is much useful that this function have the same name prefixed with '_' underscore.

Reply:
It is logic to have `rpm` in the name of function because it only creates rpm repos, however if you want to just underscore it is `createrepo` caller I'm fine with it, then maybe `_createrepo` is enough, no need in `_do` prefix.

Reply:
renamed to `_createrepo`.